airflow_kubernetes_job_operator/kubernetes_job_operator.py

```diff
@@ -236,6 +244,18 @@ def _create_kubernetes_job_name_prefix(cls, name):
     def _to_kubernetes_env_list(cls, envs: dict):
         return [{"name": k, "value": f"{envs[k]}"} for k in envs.keys()]
 
+    @classmethod
+    def _to_kubernetes_volumes_list(cls, mount_files_from_secret: dict):
+        result_list = []
+        for secret_name, v in mount_files_from_secret.items():
+            for secret_key, path_to_file in v.items():
+                result_list.append({
+                    "name": secret_name,
```

Comment:
See below the secret name in the spec volume generation. This name must be generated with some prefix as to no override other secrets coming from the template.

Reply:
Oh, good idea

Reply:
> This name must be generated with some prefix as to no override other secrets coming from the template.

This name we get from the input dictionary and it is [unique](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/) in that namespace. So if somebody add this name to the template, secret will be rewritten.
I propose to add some info about it to the readme.
How about it?

Reply:
Hum... but the name is not the secret. That is the final yaml would look:
```yaml
volumes:
    - name: [dependent on the deployment and can be multiple pointing to the same secret]
      secret:
        secretName: [always the sname]
```

e.g. the volume name cannot be the secret name.

Reply:
`volumes:
    - name: [dependent on the deployment and can be multiple pointing to the same secret]
      secret:
        secretName: [always the sname]`

Yep, it is true. 
But secretName must be unique always, so I don't understand why use for volume not secret name :)

An usage of the `__kube_op_` or other prefix - not helps, because somebody do this naming sooner or later :)

Reply:
I seem to be missing something. From the link, the pod volume mounts are as follows:
```yaml
apiVersion: v1
kind: Secret
metadata:
  name: dotfile-secret
data:
  .secret-file: dmFsdWUtMg0KDQo=
---
apiVersion: v1
kind: Pod
metadata:
  name: secret-dotfiles-pod
spec:
  volumes:
    - name: secret-volume
      secret:
        secretName: dotfile-secret
  containers:
    - name: dotfile-test-container
      image: registry.k8s.io/busybox
      command:
        - ls
        - "-l"
        - "/etc/secret-volume"
      volumeMounts:
        - name: secret-volume
          readOnly: true
          mountPath: "/etc/secret-volume"
```

How would yours be loaded?

Also it seems I need to add a print yaml mode to the operator to allow to print the final yaml result so we'd have less confusion. Hum... :)